en-US/Conductors_Companion/03_Preparing_Quests_and_Adventures/Combat_Encounter_Building/Scaling_Monster_Combinations_4_Characters.md

```diff
@@ -0,0 +1,25 @@
+<!-- markdownlint-disable-next-line no-emphasis-as-heading -->
+_Four Characters (Hard Challenges)_
+
+| Character Level | 1 Monster of same CR | 2 Monsters of same CR | 4 Monsters of same CR | 6 Monsters of same CR | 8 Monsters of same CR | 12 Monsters of same CR | 1 Boss + 2 Monsters | 1 Boss + 3 Monsters | 1 Boss + 4 Underlings | 1 Boss + 2 Lieutenants + 8 Minions |
```

Comment:
Since I can only have a single header line, I had to combine the two that were used here. It's not ideal and there may be ways to make this table easier to read (and those for other numbers of characters as well).

<img width="492" height="175" alt="The current rendering of this header" src="https://github.com/user-attachments/assets/64932bcd-012d-4126-919c-7b988ce3ab0b" />

Also, the terms "Underling" and "Lieutenant" are not used outside of these tables, so we should explain them. (More about that in the Google Doc.)